config.py

```diff
@@ -1103,16 +1215,20 @@ def get_logging_config(cfg_path=None):
         'syslog_port': None,
     }
     if system_os == 'windows':
-        logging_config['windows_collector_log_file'] = os.path.join(_windows_commondata_path(), 'StackState', 'logs', 'collector.log')
-        logging_config['windows_forwarder_log_file'] = os.path.join(_windows_commondata_path(), 'StackState', 'logs', 'forwarder.log')
-        logging_config['windows_dogstatsd_log_file'] = os.path.join(_windows_commondata_path(), 'StackState', 'logs', 'dogstatsd.log')
+        logging_config['collector_log_file'] = os.path.join(_windows_commondata_path(), 'StackState', 'logs', 'collector.log')
+        logging_config['forwarder_log_file'] = os.path.join(_windows_commondata_path(), 'StackState', 'logs', 'forwarder.log')
+        logging_config['dogstatsd_log_file'] = os.path.join(_windows_commondata_path(), 'StackState', 'logs', 'dogstatsd.log')
         logging_config['jmxfetch_log_file'] = os.path.join(_windows_commondata_path(), 'StackState', 'logs', 'jmxfetch.log')
+        logging_config['service_log_file'] = os.path.join(_windows_commondata_path(), 'StackState', 'logs', 'service.log')
+        logging_config['log_to_syslog'] = False
     else:
         logging_config['collector_log_file'] = '/var/log/stackstate/collector.log'
         logging_config['forwarder_log_file'] = '/var/log/stackstate/forwarder.log'
-        logging_config['dogstatsd_log_file'] = '/var/log/stackstate/stsstatsd.log'
+        logging_config['dogstatsd_log_file'] = '/var/log/stackstate/dogstatsd.log'
```

Comment:
maybe we need to change dogstatsd to stsstatsd.

Reply:
done